packages/wdio-electron-service/src/capabilities.ts

```diff
@@ -22,7 +22,7 @@ const isElectron = (cap: unknown) => (cap as WebdriverIO.Capabilities)?.browserN
 /**
  * Get capability independent of which type of capabilities is set
  */
-export function getElectronCapabilities(caps: Capabilities.RemoteCapability) {
+export function getElectronCapabilities(caps: Capabilities.TestrunnerCapabilities) {
```

Comment:
Given that we already abstract caps from possible multiremote capabilities, this type should be
```suggestion
export function getElectronCapabilities(caps: Capabilities.RequestedStandaloneCapabilities) {
```